### PR TITLE
Added information about a portable metal-stack setup

### PR DESCRIPTION
**Pull request by mwindower**

### CODE REVIEW

File: docs/src/overview/hardware.md [hunk 1]
```diff
@@ -26,3 +26,16 @@ At the moment we support the following switch types:
...
+
+## Portable metal-stack Setup DIY
+
+A minimal physical hardware setup may contain at least the following components:
+
+|    | Vendor     | Series           | Model                | Function             |
+| :- | :--------- | :--------------- | :------------------- | :------------------- |
+| 2x | Edgecore   | AS5500 Series    | AS5512-54x (10G)     | leaf / exit switches |
+| 1x | Supermicro | Microcloud       | SYS-5039MA16-H12RFT  |                      |
+| 1x | Unifi      | Edgemax          | Edgerouter Pro       | front router for internet and out-of-band access to servers and switches |
```
Comment (Gerrit91) [inline docs/src/overview/hardware.md:38]: You need to make the first column one character wider and add a heading in order to properly render the table with Julia Markdown (don't ask me for the exact reason).

Here is a suggestion that works (including some typographical improvements):

```suggestion
| #   | Vendor      | Series           | Model                | Function                                                                 |
| :-- | :---------- | :--------------- | :------------------- | :----------------------------------------------------------------------- |
| 2x  | Edge-Core   | AS5500 Series    | AS5512-54x (10G)     | Leaf / Exit switches                                                     |
| 1x  | Supermicro  | Microcloud       | SYS-5039MA16-H12RFT  | Usable machines                                                          |
| 1x  | Unifi       | Edgemax          | Edgerouter Pro       | Front router for internet and out-of-band access to servers and switches |
```

File: docs/src/overview/hardware.md [hunk 2]
```diff
@@ -26,3 +26,16 @@ At the moment we support the following switch types:
...
+
+Besides that, a 6HE rack with 1000mm depth and a portable LTE modem is needed.
+This MVP will yield in 12 useable machines, one of them will be reserved as management server.
```
Comment (Gerrit91) [inline docs/src/overview/hardware.md:41]: A single line does not have any impact on the rendered Markdown. For a new paragraph add a double line break. Also "useable" --> "usable", here is a suggestion:

```suggestion
Besides that, a 6HE rack with 1000mm depth and a portable LTE modem is needed.

This MVP will yield in 12 usable machines, one of them will be reserved as management server.
```

**Comment by mwindower** [comment]
> Hey, I added some feedback to the changes. One general question to this... how do we know that this mini-setup will actually work? Can we ensure that go-hal is compatible with this server hardware?

it's very likely that this hardware is compatible but it's not 100% sure :-)

**Comment by Gerrit91** [comment]
Added  @majst01 to review, from my side it's fine. Would be better though if we could really guarantee that our stuff works with the proposed hardware. Otherwise someone could buy this and then we will find out that we cannot make it work. :sweat_smile:

**Comment by majst01** [comment]
I would add a hint that this should work but is currently untested.

**Pull request merged**